src/services/api/api-base.js

```diff
@@ -75,10 +76,32 @@ export function tryCastToApiError(error) {
   throw error
 }
 
+export function toUrlParam(value) {  
+  if (value instanceof(Date))  return value.toISOString();
+  if (value instanceof(Object)) return JSON.stringify(value);  
+  return value; 
+}
+
+export function toUrlParams(valueObj) {
```

Comment:
check for undefined, otherwise if undefined is passed, you are returning empty object {}

Reply:
fix in CHM271 and CHM270